Orchestrator/docs/NLRModels.md

```diff
@@ -1,19 +1,69 @@
 # Prebuilt Language Models
 
-Prebuilt language models have been trained towards more sophisticated tasks for both monolingual as well as multilingual scenarios. In public preview only English models are made available.
+Prebuilt language models have been trained towards more sophisticated tasks for both monolingual as well as multilingual scenarios,
+including intent prediction and entity extraction.
 
 ## Models Description
 The public preview of Orchestrator includes the following prebuilt language models available in [versions repository][2].
 
 ### pretrained.20200924.microsoft.dte.00.03.en.onnx
-This is a fast and small base model with sufficient accuracy but if the accuracy and not speed and memory size is critical then consider other options. It is a 3-layer pretrained BERT model optimized for conversation for example-based use ([KNN][3]).
+This is a fast and small EN-only base model for intent detection with sufficient prediction performance.
+We suggest using this model if speed and memory size is critical to your deployment environment,
+otherwise consider other options. It is a generic 3-layer pretrained
+[Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
 
 ### pretrained.20200924.microsoft.dte.00.06.en.onnx
-This is a high quality base model that strikes the balance between size, speed and accuracy. It is a 6-layer pretrained BERT model optimized for conversation for example-based use ([KNN][3]). This is the default model used if none explicitly specified.
+This is a high quality EN-only base model for intent detection that strikes the balance between size,
+speed and predictive performance.
+It is a 6-layer pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]),
+thus it can be used out of box. This is the default model used if none explicitly specified.
```

Comment:
Highlight the defaults. Suggestion: Create a top ## Default Models section and move the default models there accordingly.

Reply:
Also, need to identify those that would work on lower architecture (32bit)

Reply:
sure